### ISSUE TITLE
Alternative methods for visualizing joint trajectories

### ISSUE THREAD

**Issue by marip8** [tesseract-robotics/tesseract_ros2#126]
When a new trajectory is received by the joint trajectory monitor, it always appends the trajectory to the visualization using the event linked below.

https://github.com/tesseract-robotics/tesseract_ros2/blob/3b7e8b0174c9918089732164cc1ccf8656469580/tesseract_rviz/src/joint_trajectory_monitor_properties.cpp#L108

This can be problematic for displaying the results of online planning (as in [the example](https://github.com/tesseract-robotics/tesseract_ros2/blob/master/tesseract_ros_examples/src/online_planning_example_node.cpp)) because the computer will quickly run out of memory stashing all the trajectories.

We should consider adding a mechanism to the existing widget to allow a trajectory (maybe with the same namespace and identifier) to be overwritten rather than appended to the list of trajectories. Alternatively, we might also think about making a display specifically for joint trajectories that doesn't stash all trajectories

**Comment by rjoomen** [comment]
I agree, and overwriting when namespace and identifier match seems like the simplest and most logical solution.

**Comment by Levi-Armstrong** [comment]
This should now be addressed but you need port changes from a PR recently merged into tesseract_ros repository.

**Comment by marip8** [comment]
Linking to the [PR for reference](https://github.com/tesseract-robotics/tesseract_ros/pull/253)

**Comment by marip8** [comment]
On a related note, would it be difficult to visualize all individual states of the trajectory at once? That would also be a nice feature for the online demo so you could watch how all the states change as new trajectories are planned

**Comment by marip8** [comment]
Closing; addressed in #131
